src/main/java/frc/robot/Calibrations.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot;
+
+import edu.wpi.first.math.controller.PIDController;
+import frc.robot.Constants.SwerveModuleConstants;
+
+public final class Calibrations {
+  public static final double PLACEHOLDER_DOUBLE = 1.0;
+  public static final int PLACEHOLDER_INT = 1;
+
+  /** Controller on module speed for rotating to target Input degrees, output [0,1] */
+  public static final PIDController ROTATE_TO_TARGET_PID_CONTROLLER = new PIDController(0.030, 0, 0.000); //From 2022
+  public static final double ROTATE_TO_TARGET_FF = 0.1; //From 2022
+
+  public static final class SwerveModuleCalibrations{
+    public static final double DRIVING_P = PLACEHOLDER_DOUBLE,
+      DRIVING_I = PLACEHOLDER_DOUBLE,
+      DRIVING_D = PLACEHOLDER_DOUBLE,
+      DRIVING_FF = 1 / SwerveModuleConstants.DRIVE_WHEEL_FREE_SPEED_RPS;
+    public static final double DRIVING_MIN_OUTPUT = -1;
+    public static final double DRIVING_MAX_OUTPUT = 1;
+    
+    public static final double TURNING_P = PLACEHOLDER_DOUBLE,
+      TURNING_I = PLACEHOLDER_DOUBLE,
+      TURNING_D = PLACEHOLDER_DOUBLE,
```

Comment:
For this PID, what's the input and output units? Useful to put in a comment. FF probably has different units I think?